api/hydrophoneApi.go

```diff
@@ -133,6 +133,7 @@ func (a *Api) SetHandlers(prefix string, rtr *mux.Router) {
 	accept := rtr.PathPrefix("/accept").Subrouter()
 	accept.Handle("/signup/{confirmationid}", varsHandler(a.acceptSignUp)).Methods("PUT")
 	accept.Handle("/forgot", varsHandler(a.acceptPassword)).Methods("PUT")
+	accept.Handle("/patient/forgot", varsHandler(a.acceptPassword)).Methods("PUT")
```

Comment:
Why add a new route since it calls the same method at the end?

Reply:
the new route can be removed if we are to consider that the same route will accept 2 different types of behavior based on the payload.